### DISCUSSION TITLE
Será que o Google Analytics lançara um versão baseada em Service Workers?

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#41] [Perguntas e Respostas]
Parece algo viável até... O que vcs acham da viabilidade de algo do tipo?

**Comment by felipefialho** [comment]
Ninguém?

@frontendbr/admins

**Comment by bernardodiasc** [comment]
Seria legal dar uma contextualizada na questão. :)

**Comment by eduardojmatos** [comment]
@felquis eu acho que é bem provável, mas não sei se os produtos que o Google tem costumam ter essas tecnologias tão recentes. Esse era o foco deles nos tempos antigos (exemplo do falecido Google Wave, que foi base pra várias coisas como o auto save do Gmail, e etc).

A ideia é bem bacana, mas pensando na aplicação do Analytics, não sei se é viável ou faz sentido. Imagina que o usuário fica sem internet. Qual seria o sentido de usar o Service Workers no Analytics se o próprio site não usa essa tecnologia? Se a aplicação for full client-side e tiver suporte a offline (seja na gravação de dados ao acesso a assets cacheads) daí concordo com você que seria fantástico :)
